src/intfuncs.c

```diff
@@ -620,6 +620,293 @@ Obj FuncINT_STRING ( Obj self, Obj string )
 
 /****************************************************************************
 **
+*F SmallInt Bitmask operations
+*
+*  Since all our functions have special handlers and no BODY, NLOC is unused,
+*  so we repurpose it.
+*
+*/
+
+#ifdef SYS_IS_64_BIT
+#define LENSTART_WIDTH 8
```

Comment:
These days, we try to use enums for constants like this. They give better compiler diagnostics, and one is likely to shoot oneself into the foot, e.g. with `#define VALUE 1+2   ...  x = VALUE * 2;`.